### ISSUE TITLE
RIP: Modify Linux table bad command

### ISSUE THREAD

**Issue by lthurlow** [lthurlow/scaling-computing-machine#6]
```
[anhost.py:287 - Thread-2  modify_linux_tables]      DEBUG MODIFY_LINUX_TABLES
[anhost.py:308 - Thread-2  modify_linux_tables]      DEBUG Adding Linux Route: {'Use': '0', 'Iface': 'eth2', 'Metric': '0', 'Destination': '10.0.1.0', 'Genmask': '255.255.255.0', 'Flags': 'U', 'TTL': '2015196034058505028', 'Ref': '0', 'Gateway': '0.0.0.0'}
SIOCADDRT: Invalid argument
[anhost.py:311 - Thread-2  modify_linux_tables]       INFO set route returned with exit status: 7
[anhost.py:308 - Thread-2  modify_linux_tables]      DEBUG Adding Linux Route: {'Use': '0', 'Iface': 'eth2', 'Metric': '1', 'Destination': '10.0.2.0', 'Genmask': '255.255.255.0', 'Flags': 'U', 'TTL': '2015196034058505028', 'Ref': '0', 'Gateway': '10.0.1.3'}
SIOCADDRT: Network is unreachable
[anhost.py:311 - Thread-2  modify_linux_tables]       INFO set route returned with exit status: 7
[anhost.py:308 - Thread-2  modify_linux_tables]      DEBUG Adding Linux Route: {'Use': '0', 'Iface': 'eth1', 'Metric': '0', 'Destination': '10.0.0.0', 'Genmask': '255.255.255.0', 'Flags': 'U', 'TTL': '2015196034023437908', 'Ref': '0', 'Gateway': '0.0.0.0'}
SIOCADDRT: Invalid argument
[anhost.py:311 - Thread-2  modify_linux_tables]       INFO set route returned with exit status: 7
```

**Comment by lthurlow** [comment]
```
[anhost.py:287 - Thread-2  modify_linux_tables]      DEBUG MODIFY_LINUX_TABLES
[anhost.py:308 - Thread-2  modify_linux_tables]      DEBUG Adding Linux Route: {'Use': '0', 'Iface': 'eth2', 'Metric': '0', 'Destination': '10.0.1.0', 'Genmask': '255.255.255.0', 'Flags': 'U', 'TTL': '2015196034058505028', 'Ref': '0', 'Gateway': '0.0.0.0'}
SIOCADDRT: Invalid argument
[anhost.py:311 - Thread-2  modify_linux_tables]       INFO set route returned with exit status: 7
[anhost.py:308 - Thread-2  modify_linux_tables]      DEBUG Adding Linux Route: {'Use': '0', 'Iface': 'eth2', 'Metric': '1', 'Destination': '10.0.2.0', 'Genmask': '255.255.255.0', 'Flags': 'U', 'TTL': '2015196034058505028', 'Ref': '0', 'Gateway': '10.0.1.3'}
SIOCADDRT: Network is unreachable
[anhost.py:311 - Thread-2  modify_linux_tables]       INFO set route returned with exit status: 7
[anhost.py:308 - Thread-2  modify_linux_tables]      DEBUG Adding Linux Route: {'Use': '0', 'Iface': 'eth1', 'Metric': '0', 'Destination': '10.0.0.0', 'Genmask': '255.255.255.0', 'Flags': 'U', 'TTL': '2015196034023437908', 'Ref': '0', 'Gateway': '0.0.0.0'}
SIOCADDRT: Invalid argument
[anhost.py:311 - Thread-2  modify_linux_tables]       INFO set route returned with exit status: 7
```

**Comment by lthurlow** [comment]
should be fixed in: f617932520f6c2b23d12fe692e9cc9ee34000d04, changed core logic to tables, as well as mutexes when touch route files or receiver updates.
